tests/clients/test_experiment.py

```diff
@@ -92,6 +97,37 @@
     "endedAt": RUN_ENDED_AT_STRING,
     "deletedAt": DELETED_AT_STRING,
 }
+EXPERIMENT_RUN_TAG = ExperimentRunTag(key="tag-key", value="tag-value")
+EXPERIMENT_RUN_TAG_BODY = {"key": "tag-key", "value": "tag-value"}
+
+OTHER_EXPERIMENT_RUN_TAG = ExperimentRunTag(
+    key="other-tag-key", value="tag-value"
+)
+OTHER_EXPERIMENT_RUN_TAG_BODY = {"key": "other-tag-key", "value": "tag-value"}
+
+EXPERIMENT_RUN_PARAM = ExperimentRunParam(
+    key="parameter-key", value="tag-value"
+)
+EXPERIMENT_RUN_PARAM_BODY = {"key": "parameter-key", "value": "tag-value"}
+
+
```

Comment:
nit: Remove one of the empty lines here?

Reply:
👍